app/src/main/java/org/simple/clinic/ReleaseClinicApp.kt

```diff
@@ -15,6 +18,12 @@ class ReleaseClinicApp : ClinicApp() {
     listOf(MixpanelAnalyticsReporter(this).swallowErrors())
   }
 
+  override fun onCreate() {
+    super.onCreate()
+    val logger = LoggerFactory.getLogger("com.spotify.mobius.ControllerStateBase") as Logger
+    logger.level = Level.OFF
```

Comment:
Can we leave a comment here which explains why we're disabling this logging explicitly?

Reply:
Done 👍🏽